docs/application/native/guides/ui/efl/wearable/component-circle-genlist.md

```diff
@@ -49,6 +49,28 @@ eext_rotary_object_event_activated_set(circle_genlist, EINA_TRUE);
 
 If the second parameter is `EINA_TRUE`, the circle genlist can receive rotary events.
 
+## Title and Padding Styles for Top and Bottom Space
+
+As circle genlist only selectable in center focused item, to allow all your items to be selectable,
```

Comment:
@everLEEst 
A slight change in the structure of the sentence for better comprehension.  


The circle genlist is only selectable for an item that is center focused. To allow all the items to be selectable,  you must add `title` and `padding` style item in the top and the bottom position: